### PR TITLE
added documentation for export. issue #311

### PR DESCRIPTION
**Pull request by kay-is**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2150,6 +2150,61 @@ <h3 id="operators-partial">Partial Application, Operators as Functions</h3>
...
+
+
+
+export 123
+
+export value
+
+export num:123
+
+export func:->
```
Comment (vendethiel) [inline index.html]: the prefered syntax is generally `export func = ->`

File: index.html [hunk 2]
```diff
@@ -2150,6 +2150,61 @@ <h3 id="operators-partial">Partial Application, Operators as Functions</h3>
...
+
+export 
+  a:1
```
Comment (vendethiel) [inline index.html]: please fix the spacing

File: index.html [hunk 3]
```diff
@@ -2150,6 +2150,61 @@ <h3 id="operators-partial">Partial Application, Operators as Functions</h3>
...
+        <div class="example-ls">
+<pre class="prettyprint lang-ls">
+
+
+
+export 123
+
+export value
+
+export num:123
```
Comment (vendethiel) [inline index.html]: I'd remove this example

File: index.html [hunk 4]
```diff
@@ -2150,6 +2150,61 @@ <h3 id="operators-partial">Partial Application, Operators as Functions</h3>
...
+      <h3 id="operators-export">export</h3>
+      <p>By using the <code>export</code> operator instead of <code>exports</code> you get a more concise way to define modules.</p>
+
+      <div class="example">
+        <div class="example-ls">
+<pre class="prettyprint lang-ls">
+
+
+
+export 123
```
Comment (vendethiel) [inline index.html]: that seems to be an useless example

File: index.html [hunk 5]
```diff
@@ -2150,6 +2150,61 @@ <h3 id="operators-partial">Partial Application, Operators as Functions</h3>
...
+
+export value
```
Comment (vendethiel) [inline index.html]: could you add comma-separated values here, please?

File: index.html (hunk 2 above)
Comment (kay-is) [inline index.html]: to what?

I just tried to line it up with the JS output

File: index.html (hunk 4 above)
Comment (kay-is) [inline index.html]: Why? Is the output undefined? I found it rather strange, so I documented it.

File: index.html (hunk 2 above)
Comment (vendethiel) [inline index.html]: `a: 1`

File: index.html (hunk 4 above)
Comment (vendethiel) [inline index.html]: what's the point of exporting 123?

File: index.html (hunk 2 above)
Comment (kay-is) [inline index.html]: The example was 

    export 
      a:1
      b:->

should I change it to 

    export a:1 b:->

?

Comment (vendethiel) [inline index.html]: No, just fix the spacing after the color

Comment (kay-is) [inline index.html]: okay, got it

**Comment by kay-is** [comment]
*ping*

**Comment by vendethiel** [comment]
Thanks! Sorry, github doesn't seem to send notifications for pushes.

**Comment by kay-is** [comment]
Good to know. :)

**Pull request merged**